### PR TITLE
Add etcd operator binding example

### PR DESCRIPTION
**Pull request by akashshinde**
This adds Etcd operator binding example which makes use of auto detecting environment variable for binding.

### CODE REVIEW

File: examples/nodejs_etcd_operator/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ spec:
+  size: 5
+  version: "3.2.13"
+ ```
+3. Deploy application which uses Etcd client lib using openshift 4 devconsole.
+Test application : https://github.com/akashshinde/node-todo.git
+![](https://i.imgur.com/WGQZ1nj.png)
+
+4. Create SBR.
+```
```
Comment (otaviof) [inline examples/nodejs_etcd_operator/README.md]: Could you add [syntax highlight](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for `yaml`?

File: examples/nodejs_etcd_operator/service-binding-request.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+apiVersion: apps.openshift.io/v1alpha1
```
Comment (otaviof) [inline examples/nodejs_etcd_operator/service-binding-request.yaml]: Do we need to keep those extra `yaml` files around? Would having only the contents in the example markdown file be sufficient?

File: examples/nodejs_etcd_operator/namespace.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (otaviof) [inline examples/nodejs_etcd_operator/namespace.yaml]: Do we need to keep the namespace file in the example directory?

Comment (pmacik) [reply examples/nodejs_etcd_operator/namespace.yaml]: We are not, the namespace creation/deletion is covered already in the example commons.

Comment (akashshinde) [reply examples/nodejs_etcd_operator/namespace.yaml]: thanks @pmacik for clarifying

File: examples/nodejs_etcd_operator/service-binding-request.yaml (hunk 2 above)
Comment (akashshinde) [reply examples/nodejs_etcd_operator/service-binding-request.yaml]: having everything in `readme` does make sense. I can remove all unwanted yamls

File: examples/nodejs_etcd_operator/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+## Binding an Imported app to an In-cluster Operator Managed Etcd Database.
+
+1. Install Etcd operator using operator hub, 
+   follow https://operatorhub.io/operator/etcd
+2. Create a Etcd cluster.
```
Comment (Avni-Sharma) [inline examples/nodejs_etcd_operator/README.md]: nit pick `an etcd cluster`

Comment (akashshinde) [reply examples/nodejs_etcd_operator/README.md]: resolved https://github.com/redhat-developer/service-binding-operator/pull/205/commits/7fcfc5049f71f9652eea8328f3f6f89bcce76aae

**Pull request merged**